lib/actions/options/read-write.ts

```diff
@@ -18,6 +17,10 @@
     option: Option<Underlying, Collateral>,
     amount: MonetaryAmount<Underlying>
   ): Promise<void>;
+
+  create<Underlying extends Currency, Collateral extends Currency>(
```

Comment:
The create function should go into the "OptionPairFactory" or so. The option/obligation are created as a pair and cannot be created individually. https://github.com/interlay/xopts/blob/dev/contracts/OptionPairFactory.sol#L94